hoot-core/src/main/cpp/hoot/core/util/GeometryUtils.cpp

```diff
@@ -305,7 +305,8 @@ Envelope GeometryUtils::envelopeFromConfigString(const QString boundsStr)
   if (boundsStr.contains(","))
   {
     const QStringList bboxParts = boundsStr.trimmed().split(",");
-    if (bboxParts.size() == 4)
+    int size = bboxParts.size();
+    if (size == 4 || size == 5)
```

Comment:
Expanded this function to parse `bbox` strings that have a fifth (unnecessary) element.  It is just ignored and the first four are parsed.